src/popup/popup.ts

```
@@ -94,4 +133,156 @@ document.addEventListener('DOMContentLoaded', () => {
+      case 'show-example-url':
+        profileUrlInput.value = 'https://x.com/username';
+        profileUrlInput.focus();
+        hideError();
+        break;
+        
+      case 'open-permissions':
+        chrome.tabs.create({ url: 'chrome://extensions/?id=' + chrome.runtime.id });
+        break;
+        
+      case 'dismiss':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)